com.unity.probuilder/Content/Resources/Materials/FacePicker.mat

```diff
@@ -21,4 +21,4 @@ Material:
     m_TexEnvs: []
     m_Floats: []
     m_Colors:
-    - _Tint: {r: 1, g: 1, b: 1, a: 1}
+    - _Tint: {r: 0, g: 0, b: 0, a: 1}
```

Comment:
Is this an intended change?

Reply:
It changes based on what version of unity it's opened in, but it can't be ignored in git.